pages/customer-type.js

```diff
@@ -35,7 +36,7 @@ const CustomerType = (props) => (
           <Checkbox
             name="supply_other"
             id="customer_type_supply_other"
-            value="It will supply food to other businesses to process, sell or serve"
```

Comment:
one thing that occurs to me is that this change (in general) will break any half-complete registrations!

Reply:
will catch in testing